aries-site/src/data/wcag/main.json

```diff
@@ -0,0 +1,512 @@
+[
+    {
+      "rule": "1.1.1",
+      "label": "Non-text content",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.1",
+      "label": "Audio-only and Video-only (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.2",
+      "label": "Captions (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.3",
+      "label": "Audio Description or Media Alternative (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.4",
+      "label": "Captions (Live)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.5",
+      "label": "Audio Description (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.6",
+      "label": "Sign Language (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.7",
+      "label": "Extended Audio Description (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.8",
+      "label": "Media Alternative (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.9",
+      "label": "Audio-only (Live)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.3.1",
+      "label": "Info and Relationships",
+      "status": "passed",
+      "notes": ""
+    },
+    {
+      "rule": "1.3.2",
+      "label": "Meaningful Sequence",
+      "status": "not-applicable",
```

Comment:
@jcfilben should this be conditional?

Reply:
I feel like the sequence things are read out in is not controlled directly by the main component so I would leave it as not-applicable

Reply:
ah okay that being said then should we change the Page component 
https://design-system.hpe.design/components/page#wcag-compliance 
for this rule to be `not-applicable`

Reply:
I could go either way on page, this rule is more related to the contents of the page but I felt like it was a nice thing to call out for people to be aware of as they at working at the page level so I was leaning towards flagging as conditional there. Its subjective though, open to other thoughts

Reply:
I see your point the way I looked at it was the same as Page where Main ensures that whatever content is inside of it is the primary content of the page. So then if the children inside Main are rendered in a logical DOM order kinda the same as with Page.

Reply:
That makes sense, I would be open to marking this as conditional here too

Reply:
okay Im also open to either but I think if we decide conditional it should apply to both Main and Page or `not-applicable` should be for both as well